### PR TITLE
SPLAT-728: [wip] introduce periodic e2e for aws localzones

### PR DESCRIPTION
**Pull request by rvanderp3**
Introduces a new, weekly, periodic which runs e2e tests on an AWS localzone cluster.  Chains are based on `ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/network-mtu/localzone/cucushift-installer-rehearse-aws-ipi-network-mtu-localzone-workflow.yaml`

### CODE REVIEW

**Comment by rvanderp3** [comment]
/uncc @bradmwilliams 
/uncc @danilo-gemoli 

uncc'ing to reduce noise while we verify the workflow

**Comment by rvanderp3** [comment]
cc: @mtulio

**Comment by rvanderp3** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.13-e2e-aws-localzone-ovn

File: ci-operator/config/openshift/release/openshift-release-master__nightly-4.13.yaml [hunk 1]
```diff
@@ -659,6 +659,16 @@ tests:
       enable:
       - observers-resource-watch
     workflow: openshift-e2e-aws-sdn
+- as: e2e-aws-localzone-ovn
+  interval: 168h
+  steps:
+    cluster_profile: aws-2
```
Comment (mtulio) [inline ci-operator/config/openshift/release/openshift-release-master__nightly-4.13.yaml:665]: @rvanderp3 can we change as we had the confirmation?:

```suggestion
    cluster_profile: aws-3
```

File: ci-operator/jobs/openshift/release/openshift-release-master-periodics.yaml [hunk 2]
```diff
@@ -42933,6 +42933,82 @@ periodics:
...
+  decoration_config:
+    skip_cloning: true
+  extra_refs:
+  - base_ref: master
+    org: openshift
+    repo: release
+  interval: 168h
+  labels:
+    ci-operator.openshift.io/cloud: aws
+    ci-operator.openshift.io/cloud-cluster-profile: aws-2
```
Comment (mtulio) [inline ci-operator/jobs/openshift/release/openshift-release-master-periodics.yaml:42948]:
```suggestion
    ci-operator.openshift.io/cloud-cluster-profile: aws-3
```

File: ci-operator/jobs/openshift/release/openshift-release-master-periodics.yaml [hunk 3]
```diff
@@ -42933,6 +42933,82 @@ periodics:
...
+        items:
+        - key: credentials
+          path: credentials
+        secretName: boskos-credentials
+    - name: ci-pull-credentials
+      secret:
+        secretName: ci-pull-credentials
+    - name: cluster-profile
+      secret:
+        secretName: cluster-secrets-aws-2
```
Comment (mtulio) [inline ci-operator/jobs/openshift/release/openshift-release-master-periodics.yaml:43005]:
```suggestion
        secretName: cluster-secrets-aws-3
```

File: ci-operator/config/openshift/release/openshift-release-master__nightly-4.13.yaml [hunk 4]
```diff
@@ -659,6 +659,16 @@ tests:
...
+    env:
+      BASE_DOMAIN: aws-2.ci.openshift.org
```
Comment (mtulio) [inline ci-operator/config/openshift/release/openshift-release-master__nightly-4.13.yaml:667]:
```suggestion
      BASE_DOMAIN: aws-3.ci.openshift.org
```

**Review by mtulio — COMMENTED** [review]
's/aws-2/aws-3/g'
https://redhat-internal.slack.com/archives/CBN38N3MW/p1676276703894429?thread_ts=1676052953.814869&cid=CBN38N3MW

**Comment by mtulio** [comment]
@rvanderp3 can we replace the target card to [SPLAT-728](https://issues.redhat.com/browse/SPLAT-728)? I thing it describes exactly the goal we need to achieve here. Wdyt?

**Comment by rvanderp3** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.13-e2e-aws-localzone-ovn

**Comment by rvanderp3** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.13-e2e-aws-localzone-ovn

**Comment by mtulio** [comment]
Hey @rvanderp3 , can we close PR card as dup of https://github.com/openshift/release/pull/36911 ?

**Pull request closed without merging**